### ISSUE TITLE
Warning: Accessing non-existent property (Node 14)

### ISSUE THREAD

**Issue by friederbluemle** [electrode-io/electrode-native#1558]
When using Node.js 14, we see tons of warnings printed to the console, e.g.:

```
Accessing non-existent property 'chmod' of module exports inside circular dependency
```

Investigate what needs to be done to fix/suppress them.

**Comment by friederbluemle** [comment]
Same with Node.js 14.1.0

See https://github.com/nodejs/node/issues/32987 for more info.

**Comment by hoangman1108** [comment]
i have same problem with node 14

**Comment by friederbluemle** [comment]
Turns out, this was quite easy to fix by updating `shelljs` to `0.8.4`: #1647
